ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -42,7 +42,8 @@ export class MigrationProjectService extends AbstractService {
 
         if (event.isTypeOf(NewExecutionStartedEvent)) {
             monitoredProject.executions.push((<NewExecutionStartedEvent>event).execution);
-        } else if (event.isTypeOf(ExecutionUpdatedEvent)) {
+        }
```

Comment:
As I wrote above "This is not a normal PR, please do not comment what I didn't" - by which I mean, I pinpointed those which I suggest to discuss and merge. The rest is just rebase garbage.